official/vision/image_classification/resnet_imagenet_main.py

```diff
@@ -182,15 +185,13 @@ def run(flags_obj):
     if dtype == 'float16':
       # TODO(reedwm): Remove manually wrapping optimizer once mixed precision
       # can be enabled with a single line of code.
-      optimizer = tf.keras.mixed_precision.experimental.LossScaleOptimizer(
-          optimizer, loss_scale=flags_core.get_loss_scale(flags_obj,
-                                                          default_for_fp16=128))
-    if flags_obj.automatic_mixed_precision:
-        if dtype == 'float16':
-            raise RuntimeError("Automatic mixed precision should not be called in conjunction with "
-                               "other types of mixed precision training. Set --dtype=fp32 instead.")
-        optimizer = tf.compat.v1.train.experimental.enable_mixed_precision_graph_rewrite(optimizer)
-        
+      if flags_dict["fp16_implementation"] == "graph_rewrite":
+          optimizer = tf.compat.v1.train.experimental.enable_mixed_precision_graph_rewrite(optimizer)
```

Comment:
This should not be compat.v1, we worked really hard to get it back to experimental from purely deprecated.

Reply:
nice, thanks. Changing this to `tf.train.experimental.enable_mixed_precision_graph_rewrite`